nextstrain/cli/util.py

```diff
@@ -808,3 +808,9 @@ def __init__(self, version: str, *, compliant: bool, original: str = None):
         super().__init__(version)
         self.compliant = compliant
         self.original = original if original is not None else version
+
+
+T = TypeVar("T")
+
+def uniq(xs: Iterable[T]) -> Iterable[T]:
+    return dict.fromkeys(xs).keys()
```

Comment:
Maybe worth noting that a crucial feature of this helper function is that order is preserved? I remembered Python started preserving dict order at some point, and had to look up that it was starting with 3.7 which is no problem for CLI which requires at least 3.8.

Reply:
Ah, sure. Will do.

FWIW, CPython has preserved dict order (as an implementation detail, but one that lots of code relied on) for quite a bit longer than the language spec has.